docs/gateway-api.md

```diff
@@ -2,7 +2,7 @@
 
 ## Add or update a backend
 
-```$xslt
+```bash
```

Comment:
should it be `shell` instead .. I am not sure what mkdocs suggests

Reply:
if shell works I would prefer that

Reply:
looks like just `sh` might also be valid.. lets find out what is best and do that consistently.

For example interesting link .. not necessarily for mkdocs but still useful

https://pawamoy.github.io/markdown-exec/usage/shell/#exec-76--markdown

Reply:
Looks like `shell` is an available lexer for `mkdocs-material`: https://pygments.org/docs/lexers/#lexers-for-various-shells

Based on the lexer name and the relevant short names I think they all point to the same thing under the hood:
<img width="793" alt="image" src="https://github.com/trinodb/trino-gateway/assets/10821555/91af76e0-7085-41d4-8d10-ad4148cc97e0">

I'll try it on my local env in a bit and push if it renders

Reply:
I just tried as well. Lets see what is most commly used in the Trino docs (even though its sphinx vs mkdocs)

Reply:
All three definitely look the same

Reply:
Okay .. Trino docs use `shell` .. lets go with the same

Reply:
looks like there's 13 instances of `bash` and 18 of `shell` 😄 
```
trino % grep -nr '```bash' docs | wc -l
      13
trino % grep -nr '```sh' docs | wc -l
      18
trino % grep -nr '```shell' docs | wc -l
      18
```

All occurences of `shell` are part of the `sphinx` docs and all occurences of `bash` are in the `docs/README.md` file